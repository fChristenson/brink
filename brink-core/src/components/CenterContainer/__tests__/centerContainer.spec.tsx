import React from "react";
import { CenterContainer, styles } from "../";
import { childrenTests } from "../../../libs/tests/childrenTest";
import { baseTests } from "../../../libs/tests/baseTests";

describe("Center container", () => {
  baseTests(<CenterContainer />, styles);
  childrenTests(<CenterContainer />);
});
