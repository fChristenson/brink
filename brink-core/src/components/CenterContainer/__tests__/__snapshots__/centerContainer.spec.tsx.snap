// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Center container Base test should assert that the component matches the snapshot 1`] = `"<div style=\\"display:flex;justify-content:center;align-items:center\\"></div>"`;

exports[`Center container Base test should assert that the component matches the snapshot 2`] = `
Object {
  "alignItems": "center",
  "display": "flex",
  "justifyContent": "center",
}
`;
